docs/docs/integrations/project-management/github.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Should this be under project management? Not sure if that's the right category

Reply:
Yes, I think so, as it helps to see the status of your features in your GitHub project(Repo).